### PR TITLE
Allow Attendees_Table::__construct accept arguments [RC]

### PR DESCRIPTION
**Pull request by aaemnnosttv**
This is a resubmit of #57 for possible inclusion in `release/119`, per @zbtirrell.

### CODE REVIEW

**Comment by zbtirrell** [comment]
Internal reference, this is logged as Central ticket [#36286](https://central.tri.be/issues/36286) (Next Release).

File: src/Tribe/Tickets/Attendees_Table.php [hunk 1]
```diff
@@ -14,9 +14,18 @@ class Tribe__Events__Tickets__Attendees_Table extends WP_List_Table {
...
 	 * Class constructor
+	 *
+	 * @param array $args  additional arguments/overrides
+	 *
+	 * @see WP_List_Table::__construct()
 	 */
-	function __construct() {
-		parent::__construct( array( 'singular' => 'attendee', 'plural' => 'attendees', 'ajax' => true ) );
+	function __construct( $args = array() ) {
+		$args = wp_parse_args( $args, array(
```
Comment (bordoni) [inline src/Tribe/Tickets/Attendees_Table.php]: It might be a good idea to throw a filter there, what do you think @borkweb?

Comment (borkweb) [inline src/Tribe/Tickets/Attendees_Table.php]: :+1: Yeah, I like that idea. Could you filter the args after the `wp_parse_args` call with a filter name of: `tribe_events_tickets_attendees_table_args'?

Comment (aaemnnosttv) [inline src/Tribe/Tickets/Attendees_Table.php]: Sure can, I'll add that

**Comment by bordoni** [comment]
It's good for a merge.

**Comment by borkweb** [comment]
Thanks @aaemnnosttv!

**Pull request merged**